teamscale-gradle-plugin/MIGRATION_35.md

```diff
@@ -0,0 +1,80 @@
+# Migration guide for the `com.teamscale` Gradle plugin version 34 to 35
+Previously, the plugin provided a way to specify the partitions either globally for a project or on a per task level.
```

Comment:
It would be great if you not only mention what the previous state was but also what the new situation looks like.